udata_recommendations/tasks.py

```diff
@@ -0,0 +1,16 @@
+from flask import current_app
+
+from udata.tasks import job
+
+from .commands import process_sources, log as commands_log
+
+
+log = commands_log
+
+
+@job("add-recommendations")
+def run_add_recommendations(self, should_clean=True):
+    should_clean = should_clean in [True, 'true', 'True']
```

Comment:
I've discovered that arguments to jobs are parsed as strings, hence this line

Reply:
🙈